app/services/api/conversions/create_project_service.rb

```diff
@@ -1,26 +1,51 @@
 class Api::Conversions::CreateProjectService
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveRecord::AttributeAssignment
+
   class ProjectCreationError < StandardError; end
 
+  attribute :urn
+  attribute :incoming_trust_ukprn
+  attribute :advisory_board_date
+  attribute :advisory_board_conditions
+  attribute :provisional_conversion_date
+  attribute :directive_academy_order
+  attribute :created_by_email
+  attribute :created_by_first_name
+  attribute :created_by_last_name
+
+  validates :urn, presence: true, urn: true
+  validate :establishment_exists
+  validates :incoming_trust_ukprn, presence: true, ukprn: true
+  validate :trust_exists
+
   def initialize(project_params)
     @project_params = project_params
+    super(project_params)
   end
 
   def call
     user = find_or_create_user
-    project = Conversion::Project.new(
-      urn: @project_params[:urn],
-      incoming_trust_ukprn: @project_params[:incoming_trust_ukprn],
-      conversion_date: @project_params[:provisional_conversion_date],
-      advisory_board_date: @project_params[:advisory_board_date],
-      advisory_board_conditions: @project_params[:advisory_board_conditions],
-      directive_academy_order: @project_params[:directive_academy_order],
-      regional_delivery_officer_id: user.id
-    )
-
-    if project.save(validate: false)
-      project
+
+    if valid?
+      project = Conversion::Project.new(
+        urn: @project_params[:urn],
```

Comment:
If we don't use `@project_params` we can just do `urn: urn` here and everywhere else we try and access `@project_params`.